priv/templates/extended_bin

```diff
@@ -368,8 +368,8 @@ relx_gen_id() {
     # To prevent exhastion of atoms on target node, optionally avoid
     # generation of random node prefixes, if it is guaranteed calls
     # are entirely sequential.
-    if [ x != x$NODETOOL_NODE_PREFIX ]; then
-        echo $NODETOOL_NODE_PREFIX
+    if [ x != x"$NODETOOL_NODE_PREFIX" ]; then
```

Comment:
Why not check if `$NODETOOL_NODE_PREFIX` is defined and not empty?

Reply:
I have no strong opinions here.

Reply:
Ok, then please switch to using `-z`. That is how these are checked in the rest of the script, so good to be consistent.

Reply:
Using `-z` now.

Reply:
@tsloughter Sorry for potentially dumb question, now that you have approved the PR even though some tests have failed, what do we do next?

Reply:
Gotta figure out why the tests fail so I can merge it.

Reply:
there's sort of no reason for it; I see tests failing only on windows and this change being on the non-windows stuff?

Reply:
Is there any progress on this? Much as I would be happy to help, I have no windows knowledge myself. Can anybody help?